go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	golang.org/x/sync v0.7.0
 	golang.org/x/time v0.5.0
 	k8s.io/utils v0.0.0-20240102154912-e7106e64919e
-	sigs.k8s.io/controller-runtime v0.17.3
+	sigs.k8s.io/controller-runtime v0.18.1
```

Comment:
Please upgrade the controller runtime for the [api](https://github.com/kyma-project/lifecycle-manager/blob/c09a3bd2ca0250ca0204d3f186ac7df3fc12bd5b/api/go.mod#L9) as well.

Reply:
done.